PROSPECTUS.md

```diff
@@ -0,0 +1,25 @@
+# Kubernetes Community Days Portland Sponsorship Opportunies
+
+_Adapted from [the Bouvet Island prospectus](https://kubernetescommunitydays.org/events/2020-bouvet-island/sponsor/)_
+
+## Post-Event Social: Contact us (Limit 1)
+
+- Opportunity to sponsor a fully branded post-event social
+- 8 Tickets
```

Comment:
I want to lower the numbers of tickets; I don't want to hold most of the tickets off the market until 2 weeks before the event.  My suggestion:

Gold/Event: 5 tix, Silver: 2 tix

Reply:
Per meeting, 6 tix for event sponsorship